File: ServerScripts/testscript.py
handle = 0 # Handle to database connection

#include <db_con.py>

def doIt():
	if	ping(handle):
		print("Updating worlds name.")
		mysql_query(handle, "Update worlds set worldname='TestWorld' where w_id = 1;")
	else:
		print("Db down :(")

#Initialization:
print("Init: testscript.py")
handle = connect()
if ping(handle): # Connection open ?
	doIt()

File: ServerScripts/testscript.lua
function Init ()
	print("Hello from testscript.lua")
	--[[
	handle = mysql_init(nil)
	handle = mysql_real_connect(handle, "127.0.0.1", "root", "password", "Anthera", 3306, nil, 0)
	mysql_query(handle, "UPDATE worlds SET worldname='LUAWorld' WHERE w_id = 1;")
	mysql_close(handle)
	]]
    print("Done")
end

function Register(serverClient)
	print("Register called(Lua)!")
	print("Loginname:" .. serverClient.authData.loginname)
end